beacon-chain/node/config.go

```diff
@@ -82,3 +82,16 @@ func configureNetwork(cliCtx *cli.Context) {
 		params.OverrideBeaconNetworkConfig(networkCfg)
 	}
 }
+
+func configureInteropConfig(cliCtx *cli.Context) {
+	genStateIsSet := cliCtx.IsSet(flags.InteropGenesisStateFlag.Name)
+	genTimeIsSet := cliCtx.IsSet(flags.InteropGenesisTimeFlag.Name)
+	numValsIsSet := cliCtx.IsSet(flags.InteropNumValidatorsFlag.Name)
+	votesIsSet := cliCtx.IsSet(flags.InteropMockEth1DataVotesFlag.Name)
+
+	if genStateIsSet || genTimeIsSet || numValsIsSet || votesIsSet {
```

Comment:
Wouldn't this be `genStateIsSet && genTimeIsSet && numValsIsSet && votesIsSet`?

we need all four flags to apply for interop to work right?

Reply:
Nope anyone should be fine, technically all we need is just the correct genesis state.